test/io/testscripts.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Unit tests for main scripts (ccget, ccwrite)."""
+import os
+import unittest
+from io import StringIO
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
```

Comment:
You don't need that many blank lines between sections. Please follow the recommendations of PEP8 for formatting.

Reply:
fixed